Dockerfile

```diff
@@ -163,6 +163,9 @@ RUN cpan \
   Test::Memory::Cycle \
   Mozilla::CA
 
+RUN mkdir -p /l/local/bin
+RUN ln -s /usr/bin/unzip /l/local/bin/unzip
```

Comment:
FWIW, there shouldn't be anywhere that doesn't have `/usr/bin/unzip` -- worth removing references to `/l/local/bin` from the application side?

Reply:
I think this was when the package unzip couldn't handle 2GB+ files, so we had the local version installed. That no longer seems to be true so 👍